stm32loader/main.py

```diff
@@ -342,6 +350,7 @@ def main(*args, **kwargs):
         loader.parse_arguments(args)
         loader.connect()
         try:
+            loader.stm32.setDataTransferAndFlashPageSize(loader.configuration["family"])
```

Comment:
I think we should supply `family` to the constructor of Stm32Bootloader so that it can be used internally.

Then the configuration of data transfer and flash page sizes does not have to be called from `main.py`.